vms/vms/settings.py

```diff
@@ -63,8 +63,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'vmsadmin',
-        'PASSWORD': '0xdeadbeef',
+        'USER': '',
+        'PASSWORD': '',
```

Comment:
@smarshy I remember someone raised a issue and we thought that we will include this. I remember myself pushing this change. If you don't change it will it cause any issues in your work?